src/service/utils/layers.js

```diff
@@ -94,23 +100,31 @@ export const handleSelectQuality = (media) => {
   }
   //Replaced select with project, as select can cause errors when used with transcoders
   const source = state.Sources.selectedVideoSource
-  const mediaLayers = state.Layers.medias[source.mid].layers
-  const quality = mediaLayers.find(layer => layer.simulcastIdx === media.simulcastIdx)
+  const mediaLayers =
+    state.Layers.medias[source.mid]?.layers || state.Layers.medias['0']?.layers
```

Comment:
for better readability, I'd suggest (if it makes sense)
```suggestion
const layerIdx= source?.mid ?? '0'
 const mediaLayers = state.Layers.medias[layerIdx]?.layers
```